server/aws-lsp-identity/src/language-server/utils.ts

```diff
@@ -0,0 +1,67 @@
+import {
+    Chat,
+    CredentialsProvider,
+    IdentityManagement,
+    Logging,
+    Lsp,
+    Notification,
+    Runtime,
+    Workspace,
+    PartialInitializeResult,
+    Telemetry,
+} from '@aws/language-server-runtimes/server-interface'
+import { InitializeParams, HandlerResult, InitializeError } from 'vscode-languageserver'
+
+export function ensureSsoAccountAccessScope(scopes?: string[]): string[] {
+    const ssoAccountAccessScope = 'sso:account:access'
+
+    if (!scopes) {
+        scopes = [ssoAccountAccessScope]
+    } else if (!scopes.includes(ssoAccountAccessScope)) {
+        scopes.push(ssoAccountAccessScope)
+    }
+
+    return scopes
+}
+
+export interface ServerFeatures {
```

Comment:
This should be coming from the server interface from runtimes package for single source of truth.

Reply:
I agree and I'm happy to move it there, but I was trying to avoid changing the runtimes repo again until Eclipse/Q gives me the green light to remove clientName from the identity-management APIs.

Or are you saying this already exists in the runtimes package?  This https://github.com/aws/language-server-runtimes/blob/main/runtimes/server-interface/server.ts is the closest I found, but not usable in that form.

Reply:
Another temporary option would be something like this:
`export type ServerFeatures = Parameters<Server>[0]`, until we define interface properly in runtimes package. I would like to avoid different set of features in different places of code.

Reply:
Otherwise, please make sure to move this to runtime in the following PR.

Reply:
Added a TODO comment here to migrate this interface to runtimes